src/common/sharly_chess_config.py

```diff
@@ -44,32 +45,40 @@ def reload(cls):
         cls._stored_config = cls.load_stored_config()
 
     @staticmethod
-    def load_stored_config() -> StoredConfig:
+    def get_user_locale() -> str | None:
+        """Returns the locale used by the user at system-level,
+        if known by the i18n stuff (otherwise returns None)."""
+        if sys.platform == 'win32':  # pragma: py-not-win32
+            import locale
+            import ctypes
+
+            windll = ctypes.windll.kernel32
+            system_user_locale: str = locale.windows_locale[
+                windll.GetUserDefaultUILanguage()
+            ]
+            logger.info('User locale: %s', system_user_locale)
+            locale: str = system_user_locale[:2]
+            if locale in locales:
+                return locale
+            logger.warning('Unknown locale: %s', locale)
+            return None
+        else:
+            raise OSError(f'{sys.platform=}')
```

Comment:
```suggestion
        return None
```
If we've got that kind of OS filter, it definitely should not be here, and it should be very explicit. As we are doing whatever we can to make the app open to more OSs, I'm really opposed to having that kind of error raised. it also excludes the other windows distrib, on which SC might actually work ('cygwin' / 'msys')